index.html

```diff
@@ -209,5 +193,38 @@ <h3><img src="images/Rectangle-57.png" alt=""><span>Skills</span></h3>
             </div>
         </div>
     </section> 
+    <footer class="footer">
+        <div class="footer-content">
+            <div class="footer-text">
+                <p>
+                  I'm always<br>interested in hearing about new projects, 
+                  so if you'd like to chat please get in touch.
+                </p>
+            </div>
+            <form action="https://formspree.io/f/form" id="form" method="post">
+                <input type="text" id="name" name="fullname" placeholder="Full name" required maxlength="50"> <br><br>
+                <input type="email" id="email" name="email" placeholder="Email address">
+                <textarea name="description" id="textarea" cols="27" rows="5" required maxlength="500" placeholder="Enter text here"></textarea>
+            </form>
+            <button class="btn" type="button">Get in touch</button>
```

Comment:
- [ ] Your button should be placed inside the form and with a `type="submit"` in order for it to work. Kindly fix that.